### ISSUE TITLE
MFR/CompatThaumCraft Bug Aspect not existing

### ISSUE THREAD

**Issue by MasterTeixi** [powercrystals/MineFactoryReloaded#422]
MFR is trying to register a non existable aspect "Tempus".
That one is Added by Magic Bees, not Thaumcraft, and it wants to add it even if you don't have Magic Bees installed.

What's more, after installing it, it keeps telling "Thaumcraft" that one of it's items uses this aspect, and thaumcraft doesn't have anything to do with it, so, it crashes.

**Comment by MFernflower** [comment]
Yeah this has been really horrible for me aswell, since there's no way of turning off the Compat-Thaumcraft Module 

Edit: Report the issue here: https://github.com/skyboy/MineFactoryReloaded/issues

**Comment by Prophious** [comment]
Pllllhhheeeezzzzz fix this..... (wonders if previous version works as intended)

**Comment by MFernflower** [comment]
You could always delete the sub mod but this requires a bit of hackery

**Comment by Prophious** [comment]
but its on a server.... so no hackery here.....

**Comment by DrZed** [comment]
Skyboy add a config to disable every one of the modules :P individually
